800.renames-and-merges/b.yaml

```diff
@@ -38,7 +38,7 @@
 - { setname: bats,                     name: [bats-core,bash-bats] }
 - { setname: battery-monitor,          name: battery-monitor-devel, weak_devel: true, nolegacy: true }
 - { setname: baudline,                 name: linux-baudline, ruleset: freebsd }
-- { setname: bazarr,                   name: $0-beta, weak_devel: true, nolegacy: true }
+- { setname: bazarr,                   name: $0-beta, devel: true }
```

Comment:
`nolegacy` should stay and you don't need `devel` as long is it's versioned properly (`XbetaY`)